lib/task/tasks/composite_task.rb

```diff
@@ -0,0 +1,16 @@
+require_relative '../task'
```

Comment:
Should I prefer 'require'? I've tended to prefer `require_relative` since it shows the relative structure of the module, so for example the entire module could move to some other namespace without issue.